front/components/assistant/conversation/lib.ts

```diff
@@ -160,6 +161,13 @@ export async function submitMessage({
   );
 
   if (!mRes.ok) {
+    if (mRes.status === 413) {
```

Comment:
Where is that 413 defined? Can we use a error code/type?

Reply:
Ah that's next refusing it?

Should we bump the limit on the endpoint?

Reply:
It's indeed next refusing it. But you can apparently modify this configuration on the endpoint directly: https://stackoverflow.com/a/74338919/10716506

Reply:
Yes we do so already on other endpoints. Ithiiiiink we want to do it here as well no?